packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -13,20 +13,26 @@ import useNavigationMenu from '../use-navigation-menu';
 import useNavigationEntities from '../use-navigation-entities';
 import useConvertClassicMenu from '../use-convert-classic-menu';
 import useCreateNavigationMenu from './use-create-navigation-menu';
-import ExistingMenusOptions from './existing-menus-options';
 
 export default function NavigationMenuSelector( {
 	clientId,
 	onSelect,
 	onCreateNew,
-	canUserCreateNavigation = false,
-	canUserSwitchNavigation = false,
+	showTools = false,
```

Comment:
`tools` don't mean much to me, how about this?

```suggestion
	showNavigationManagement = false,
```

Reply:
It's the only prop that mentions the component name (i.e. `Navigation`). How about just `showManagement`?

Reply:
Renamed to `showManageActions`. Does that work?

Reply:
It does, thank you!